performance/runner/src/main.rs

```diff
@@ -0,0 +1,70 @@
+use std::{env, fs, io};
+use std::process::{Command, ExitStatus, exit};
+
+
+#[derive(Debug, Clone)]
+struct Metric<'a> {
+    name: &'a str,
+    prepare: &'a str,
+    cmd: &'a str,
+}
+
+impl Metric<'_> {
+    fn outfile(&self, project: &str) -> String {
```

Comment:
Rust newbie but does this just return the name of the results file for the specific metric?

Reply:
that's exactly what this function does 😃